multiterm/src/plugin.vala

```diff
@@ -53,6 +53,8 @@ public void plugin_init(Geany.Data data)
 	Alignment align;
 	MultiTerm.Notebook notebook;
 
+	toplevel_widgets = new List<Widget>();
```

Comment:
This probably needs to be assigned `null` in the cleanup function to avoid a leak in the GC and so there's no dangling junk left in it.

Reply:
Done.